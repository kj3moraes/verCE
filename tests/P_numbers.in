x = 12
x = 1.0
x = 0.1
x = 99
x = 1234567890.987654321
