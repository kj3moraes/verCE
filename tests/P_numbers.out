ID:x 
BECOMES:= 
NUM:12 
Handling top level expression
Parsed successfully!
ID:x 
BECOMES:= 
NUM:1.0 
Handling top level expression
Parsed successfully!
ID:x 
BECOMES:= 
NUM:0.1 
Handling top level expression
Parsed successfully!
ID:x 
BECOMES:= 
NUM:99 
Handling top level expression
Parsed successfully!
ID:x 
BECOMES:= 
NUM:1234567890.987654321 
Handling top level expression
Parsed successfully!
